Solutions/RubrikSecurityCloud/Playbooks/RubrikDataObjectDiscovery/azuredeploy.json

```diff
@@ -3,11 +3,12 @@
     "contentVersion":  "1.0.0.0",
     "metadata":  {
         "title":  "Rubrik Data Object Discovery",
-        "description":  "This playbook queries Rubrik Security Cloud to enrich the incoming event with additional information from Rubrik about the object and it’s snapshots that the event refers to.",
+        "description":  "This playbook queries Rubrik Security Cloud to enrich the incoming event with additional information from Rubrik about the object and it's snapshots that the event refers to.",
         "prerequisites":  [
             "1. The Rubrik Security Cloud data connector should be configured to send appropriate events to Microsoft Sentinel.",
             "2. The Rubrik Security Cloud solution should be configured to [connect to Rubrik Security Cloud API end points using a Service Account](https://docs.rubrik.com/en-us/saas/saas/polaris_api_access_with_service_accounts.html), the service account should be assigned a role that includes the relevant privileges necessary to perform the desired operations (see [Roles and Permissions](https://docs.rubrik.com/en-us/saas/saas/common/roles_and_permissions.html) in the Rubrik Security Cloud user guide).",
-            "3. Store Service account credentials in Key Vault and obtain keyvault name and tenantId",
+            "3. Rubrik custom connector needs to be deployed prior to the deployment of this playbook, in the same resource group and region. Relevant instructions can be found in the connector doc page.",
+            "4. Store Service account credentials in Key Vault and obtain keyvault name and tenantId",
                 "a. Create a Key Vault with unique name",
```

Comment:
Add the prerequisite playbook details here as well. Also, I don't see corresponding readme file have been updated with the prerequisite playbook details.